Java/src/main/java/Airport.java

```diff
@@ -13,107 +15,109 @@
-            } //if
-            else {
+            }
+        }
 
-            } // else
-        } //for
         return militaryPlanes;
     }
 
     public PassengerPlane getPassengerPlaneWithMaxPassengersCapacity() {
-        List<PassengerPlane> passengerPlanes = getPasPl();
+        List<PassengerPlane> passengerPlanes = getPassengerPlane();
         PassengerPlane planeWithMaxCapacity = passengerPlanes.get(0);
-        for (int i = 0; i < passengerPlanes.size(); i++) {
+
+        for (int i = 0; i < passengerPlanes.size() ; i++) {
             if (passengerPlanes.get(i).getPassengersCapacity() > planeWithMaxCapacity.getPassengersCapacity()) {
                 planeWithMaxCapacity = passengerPlanes.get(i);
             }
         }
 
-
-
-
-
-
         return planeWithMaxCapacity;
     }
 
     public List<MilitaryPlane> getTransportMilitaryPlanes() {
-    List<MilitaryPlane> transportMilitaryPlanes = new ArrayList<>();
-    List<MilitaryPlane> militaryPlanes = getMilitaryPlanes();
-    for (int i = 0; i < militaryPlanes.size(); i++) {
-    MilitaryPlane plane = militaryPlanes.get(i);
-    if (plane.getType() == MilitaryType.TRANSPORT) {
-    transportMilitaryPlanes.add(plane);
-    }
-    }
-    return transportMilitaryPlanes;
+        List<MilitaryPlane> transportMilitaryPlanes = new ArrayList<>();
+        List<MilitaryPlane> militaryPlanes = getMilitaryPlanes();
+
+        for (int i = 0; i < militaryPlanes.size() ; i++) {
+            MilitaryPlane plane = militaryPlanes.get(i);
+
+                if (plane.getType() == MilitaryType.TRANSPORT) {
+                    transportMilitaryPlanes.add(plane);
+                }
+        }
+
+        return transportMilitaryPlanes;
     }
 
     public List<MilitaryPlane> getBomberMilitaryPlanes() {
         List<MilitaryPlane> bomberMilitaryPlanes = new ArrayList<>();
         List<MilitaryPlane> militaryPlanes = getMilitaryPlanes();
-        for (int i = 0; i < militaryPlanes.size(); i++) {
+
+        for (int i = 0; i < militaryPlanes.size() ; i++) {
             MilitaryPlane plane = militaryPlanes.get(i);
+
             if (plane.getType() == MilitaryType.BOMBER) {
                 bomberMilitaryPlanes.add(plane);
             }
         }
+
         return bomberMilitaryPlanes;
 
     }
 
     public List<experimentalPlane> getExperimentalPlanes() {
         List<experimentalPlane> experimentalPlanes = new ArrayList<>();
+
         for (Plane plane : planes) {
             if (plane instanceof experimentalPlane) {
                 experimentalPlanes.add((experimentalPlane) plane);
             }
         }
+
         return experimentalPlanes;
     }
 
     public Airport sortByMaxDistance() {
         Collections.sort(planes, new Comparator<Plane>() {
-            public int compare(Plane o1, Plane o2) {
-                return o1.Get_Max_Flight_Distance() - o2.Get_Max_Flight_Distance();
+            public int compare(Plane firsPlane, Plane secondPlane) {
```

Comment:
you can use the collection sort with Comparator instead of create a compare inside the method
example:  **planes.sort(Comparator.comparingInt(Plane::getMaxFlightDistance));**
this makes an easy read method